.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: required
+dist: trusty
+language: node_js
+node_js:
+  - '8.4'
+
+script:
+  - ng build --prod
```

Comment:
can we also do a - ng test -sr

Reply:
I made the update, thanks for the reminder